### PR TITLE
add azion atlas sample

### PR DESCRIPTION
**Pull request by hendriksoares**
Add a new template to mongoDB atlas

### CODE REVIEW

File: templates/azion-atlas/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Azion Sample to connect and create API with Atlas MongoDB
+
+## Overview
+
+The Atlas MongoDB is an automation designed to deploy a database directly on the edge while enabling the configuration of creating, retrieving, updating and deleting items is a collection.
```
Comment (gabriel-azion) [inline templates/azion-atlas/README.md]:
```suggestion
The Atlas MongoDB is an automation designed to deploy a database directly on the edge while enabling the configuration of creating, retrieving, updating, and deleting items in a collection.
```

File: templates/azion-atlas/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ 
+As part of the deployment, this template creates a new GitHub repository that allows a continuous deployment workflow. This way, you're able to further configure and customize further the setup to attend to your needs, obtaining easy management and delivery of your personalized content.
```
Comment (gabriel-azion) [inline templates/azion-atlas/README.md]:
```suggestion
As part of the deployment, this template creates a new GitHub repository that allows a continuous deployment workflow. This way, you're able to further configure and customize the setup to meet your needs, obtaining easy management and delivery of your personalized content. 
```

File: templates/azion-atlas/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Any cost generated by MongoDb usage will be processed and billed separately on the MongoDB platform. Visit the [Atlas MongoDB ](https://www.mongodb.com/pricing) official documentation for more details.
+
+## Usage information
+
+**Requirements**
+ 
+Before using this template, you need to: 
+
+
+1. Create an [MongoDB account](https://www.mongodb.com/cloud/atlas/register).
```
Comment (gabriel-azion) [inline templates/azion-atlas/README.md]:
```suggestion
1. Create a [MongoDB account](https://www.mongodb.com/cloud/atlas/register).
```

**Review by gabriel-azion — APPROVED** [review]
some suggestions

**Comment by hendriksoares** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**